websites/M/Modrinth/presence.ts

```diff
@@ -0,0 +1,170 @@
+				}
+				case "mod": {
+					presenceData.state = `${
+						document.querySelector<HTMLHeadingElement>("h1.title").textContent
+					}`;
+					presenceData.buttons = [
+						{
+							label: "Open mod page",
+							url: `https://modrinth.com/mod/${pathSplit[1]}`,
+						},
+					];
+
+					switch (pathSplit[2]) {
+						case "gallery": {
+							presenceData.details = "Browsing mod gallery";
+							break;
+						}
+						case "changelog": {
+							presenceData.details = "Browsing mod changelog";
+							break;
+						}
+						case "versions": {
+							presenceData.details = "Browsing mod versions";
+							break;
+						}
+						case "settings": {
+							presenceData.details = "Browsing project settings";
+							break;
+						}
+						default: {
+							presenceData.details = "Browsing mod description";
+						}
+					}
+					break;
+				}
+				case "plugins": {
+					presenceData.details = "Looking for useful Plugins";
+					break;
+				}
+				case "resourcepacks": {
+					presenceData.details = "Looking for a beautiful Resourcepack";
+					break;
+				}
+				case "modpacks": {
+					presenceData.details = "Looking for a nice Modpack";
+					break;
+				}
+				case "notifications": {
+					presenceData.details = "Checking Notifications";
+					break;
+				}
+				case "dashboard": {
+					presenceData.details = "Looking into Dashboard";
+					break;
+				}
+				case "legal": {
+					switch (pathSplit[1]) {
+						case "terms": {
+							presenceData.details = "Reading Terms and Conditions 👀";
+							break;
+						}
+						case "privacy": {
+							presenceData.details = "Reading Privacy Policy 🔐️";
+							break;
+						}
+						case "rules": {
+							presenceData.details = "Reading Content Rules 📃";
+							break;
+						}
+						case "security": {
+							presenceData.details = "Security Notice 👮‍♂️";
+							break;
+						}
+					}
+					break;
+				}
+				default: {
+					presenceData.details = "Browsing Main page";
+				}
+			}
+			break;
+		}
+		case "docs.modrinth.com": {
+			switch (pathSplit[0]) {
+				case "docs": {
+					switch (pathSplit[1]) {
+						case "tutorials": {
+							presenceData.details = "Browsing Docs tutorials";
+							break;
+						}
+						case "details": {
+							presenceData.details = "Reading about the API";
+							break;
+						}
+						case "modpacks": {
+							presenceData.details = "Reading about modpacks";
+							break;
+						}
+						default: {
+							presenceData.details = "Browsing API information";
+							break;
+						}
+					}
+					break;
+				}
+				case "api-spec": {
+					presenceData.details = "Browsing API Documentation";
+					break;
+				}
+				default: {
+					presenceData.details = "Browsing Docs";
+				}
+			}
+			break;
+		}
+		case "blog.modrinth.com": {
+			presenceData.buttons = [
+				{
+					label: "Read blog",
+					url: "https://blog.modrinth.com",
+				},
+			];
+			if (
+				document.querySelector<HTMLHeadingElement>("h1.post-title__text") !==
+				null
```

Comment:
`document.querySelector` returns `null` or an element. `null` is falsy, so there isn't a need to use this explicit check here.
```suggestion
				document.querySelector<HTMLHeadingElement>("h1.post-title__text")
```